contracts/builder_unlock/Cargo.toml

```diff
@@ -20,10 +20,10 @@ cw-storage-plus = {version = "0.8.0", features = ['iterator']}
 schemars = "0.8.1"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 astroport-governance = { path = "../../packages/astroport-governance", default-features = false, version = "1.0.0"}
+astroport = {git = "https://github.com/astroport-fi/astroport.git", branch = "master"}
```

Comment:
Correct git path is "https://github.com/astroport-fi/astroport-core". We use main branch as default.

Reply:
Changed